### PR TITLE
PMM-13141 add release notes.

### PR DESCRIPTION
**Pull request by BupycHuk**
PMM-13141

Link to the Feature Build: SUBMODULES-0

If this PR adds or removes or alters one or more API endpoints, please review and add or update the relevant [API documents](https://github.com/percona/pmm/tree/main/docs/api) as well:

- [ ] API Docs updated

If this PR is related to some other PRs in this or other repositories, please provide links to those PRs:

- Links to related pull requests (optional).

### CODE REVIEW

File: docs/release-notes/2.43.0.md [hunk 1]
```diff
@@ -183,6 +183,8 @@ If you're looking to upgrade, you can easily [install the latest version of Perc
   
 - [PMM-13292](https://perconadev.atlassian.net/browse/PMM-13292) - Introduced an improved mechanism to resolve port conflicts when starting multiple PMM agents on the same machine, enhancing reliability and reducing manual intervention in complex monitoring setups.
 
+- [PMM-13141](https://perconadev.atlassian.net/browse/PMM-13141) - Introduced new MongoDB metrics for Feature Compatibility Version (FCV), providing insights into the MongoDB version compatibility of individual nodes, enhancing monitoring capabilities for multi-version MongoDB deployments.
```
Comment (catalinaadam) [inline docs/release-notes/2.43.0.md:186]:
```suggestion
- [PMM-13141](https://perconadev.atlassian.net/browse/PMM-13141) - Introduced new MongoDB metrics for Feature Compatibility Version (FCV),  enhancing monitoring capabilities for multi-version deployments. 

Collected via the MongoDB exporter, this metric provides insights into individual node compatibility and supports upcoming PMM advisors. 

To use this feature, enable the FCV collector with the `--enable-all-collectors` flag when adding MongoDB services to PMM.
```

Comment (BupycHuk) [reply docs/release-notes/2.43.0.md:186]: it's actually enabled by default after a comment from @igroene

Comment (catalinaadam) [reply docs/release-notes/2.43.0.md:186]: thanks @BupycHuk , I've updated my  suggestion

[the remark this answers is not in the document]
Comment (igroene) [reply docs/release-notes/2.43.0.md]: LGTM

**Pull request merged**